inference-engine/include/vpu/vpu_config.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief A header that defines common config subset for VPU plugins.
+ * Include myriad_config.hpp or hddl_config.hpp directly.
+ * These properties should be used in SetConfig() and LoadNetwork() methods of plugins
+ *
+ * @file vpu_config.hpp
+ */
+
+#pragma once
+
+#include "ie_plugin_config.hpp"
+#include "ie_api.h"
+
+#include <string>
+
+#define DECLARE_VPU_CONFIG(name) static constexpr auto name = #name
```

Comment:
Should such macro be undefined at the end of file?

Reply:
We use such macro in private config file also